### PR TITLE
docs: add changelog and readme for data plane

### PR DESCRIPTION
**Pull request by wenytang-ms**
adddataapi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ Ensure that new API versions don't break API consumers with breaking change dete
 
 ![Generate Open API specification](./media/generate-api-documentation.gif)
 
+[[Pre-Release]](#pre-release)
+API Center Data View
```
Comment (formulahendry) [inline README.md:30]: I think this feature is worthy to be a seperate section.
@sthirthala thoughts?

Comment (sthirthala) [reply README.md:30]: Yes makes sense to have separate section

Comment (sthirthala) [inline README.md:30]: Let us call it "Platform API Catalog" instead of "API Center Data View"

File: media/api-center-data-view.gif
Comment (sthirthala) [inline media/api-center-data-view.gif:1]: Instead of calling it "API Center Data View" let us call it "Platform API Catalog"

File: PRERELEASE.md [hunk 2]
```diff
@@ -3,6 +3,15 @@
 
 > Note: This changelog only includes the changes for the pre-release versions of Azure API Center for Visual Studio Code. For the changelog of stable versions, please refer to the [Changelog of Azure API Center for Visual Studio Code](https://github.com/microsoft/vscode-azureapicenter/blob/main/CHANGELOG.md).
 
+### September 9, 2024
+* Add Data View to Azure API Center:
```
Comment (sthirthala) [inline PRERELEASE.md:7]: Instead of calling it "Data View" let us call it "Platform API Catalog"

File: PRERELEASE.md [hunk 3]
```diff
@@ -3,6 +3,15 @@
...
+    * Connect to An Existing API Center from UI, Command Palette or DeepLink
+    * View and List all the APIs, Versions, Definitions
+    * Export API, Generate API Client, Generate Markdown, Open API Document from API Definitions
+    * Sign In and Sign Out from Data View
+    * Disconnect to API Center Data Plane API
```
Comment (sthirthala) [inline PRERELEASE.md:12]: Same comment as above - instead of data plane API - we can call it "Platform API Catalog"

File: PRERELEASE.md [hunk 4]
```diff
@@ -3,6 +3,15 @@
 
 > Note: This changelog only includes the changes for the pre-release versions of Azure API Center for Visual Studio Code. For the changelog of stable versions, please refer to the [Changelog of Azure API Center for Visual Studio Code](https://github.com/microsoft/vscode-azureapicenter/blob/main/CHANGELOG.md).
 
+### September 9, 2024
+* Add Data View to Azure API Center:
+    * Connect to An Existing API Center from UI, Command Palette or DeepLink
+    * View and List all the APIs, Versions, Definitions
+    * Export API, Generate API Client, Generate Markdown, Open API Document from API Definitions
+    * Sign In and Sign Out from Data View
```
Comment (sthirthala) [inline PRERELEASE.md:11]: Sign in and Sign out from "Platform API Catalog"

**Review by sthirthala — COMMENTED** [review]
Broadly speaking the main change we need to make is to replace "API Center Data View" with "Platform API Catalog"

**Comment by wenytang-ms** [comment]
all strings updated, @sthirthala please help review again, thanks

File: README.md [hunk 5]
```diff
@@ -26,6 +26,11 @@ Ensure that new API versions don't break API consumers with breaking change dete
 
 ![Generate Open API specification](./media/generate-api-documentation.gif)
 
+[[Pre-Release]](#pre-release)
+
+### Platform API Catalog
```
Comment (formulahendry) [inline README.md:31]:
```suggestion
### [[Pre-Release]](#pre-release) Platform API Catalog
```

Comment (wenytang-ms) [reply README.md]: updated

**Review by formulahendry — APPROVED** [review]

**Pull request merged**